recipes/trefide/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "trefide" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jw3132/conda-forge-tar/raw/master/trefide-{{ version }}.tar.gz
+  sha256: e462466d3a21b6c88c2958541aeaf3caf1cb2ee197f488265bf03fc848eaf562
+
+build:
+  number: 0
+  skip: true  # [win, osx]
```

I think this will always be true. You should use `win or osx` else this is probably more explicit for what you mean.

Also, assuming you need a compiler, the correct way to limit the Python version is using `skip`. Do you need to be as strict as Python 3.6 only? Or is any Python 3 version okay?

```suggestion
  skip: true  # [py2k or not linux]
```

---

Thanks. Fixed that.